docs/source/data/data_catalog.md

```diff
@@ -164,7 +164,17 @@ rockets:
     sheet_name: Sheet1
 ```
 
-Example 7: Saves an image created with Matplotlib on Google Cloud Storage
+Example 7: Loads a multisheet excel file from a local file system
```

Comment:
```suggestion
Example 7: Loads a multi-sheet Excel file from a local file system
```
:)

Reply:
Made the change in 3f493a1921cf72bd3f81bd2f49f4ec3bedecc304, thanks!